test/lightning/importer_test.exs

```diff
@@ -36,15 +36,15 @@ defmodule Lightning.ImportProjectsTest do
                 adaptor: "language-fhir",
                 enabled: true,
                 credential: "xyz",
-                body: "> fn(state => state)"
+                body: "fn(state => state)"
```

Comment:
@wanecode I changed these to match what the body actually looks like, it appears you added the prepending `>` from the YAML examples - which is a signal in YAML to create a single string from a multiline indented to the left margin of the `>`.